<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Console</class>
 <widget class="QDialog" name="Console">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/icons.qrc">
    <normaloff>:/monokuma-green.png</normaloff>:/monokuma-green.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="XLoggerWidget" name="txtConsole">
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Logging Level</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboLevels"/>
     </item>
     <item>
      <widget class="QCheckBox" name="chkWordWrap">
       <property name="text">
        <string>Word wrap</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XLoggerWidget</class>
   <extends>QTextEdit</extends>
   <header>projexui.xloggerwidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>txtConsole</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboLevels</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>Console</receiver>
   <slot>updateLogLevel(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateLogLevel(QString)</slot>
 </slots>
</ui>
